packages/core/src/data/messages/messages.json

```
@@ -12054,6 +12175,7 @@
         "MessageType_EthereumAddressOneKey": 20103,
         "MessageType_EthereumSignTxOneKey": 20104,
         "MessageType_EthereumSignTxEIP1559OneKey": 20105,
+        "MessageType_EthereumSignTxEIP7702OneKey": 20120,
```

Biome (2.1.2)

[error] The key MessageType_EthereumSignTxEIP7702OneKey was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)